### DISCUSSION TITLE
Motioneye crash when smb enabled

### DISCUSSION THREAD

**Discussion by nortuzar** [tteck/Proxmox#1443] [Help Needed]
### A clear and concise description of the issue.

Same problem as the already closed [#211 ](https://github.com/tteck/Proxmox/issues/211) on a privileged LXC. 

I enable smb/cifs feature. The problem is when turning enable the samba setting on motioneye.conf, the LXC run but the GUI wont work

cifs-utils is installed. Do I need to install samba or something?

### Please provide detailed steps to reproduce the issue.

Samba settings change to true on motioneye.conf

**Comment by JMarty** [comment]
I have the same problem.

**Comment by revnyolc** [comment]
it crashes for me too
